src/type.ml

```diff
@@ -573,7 +860,7 @@ let rec local_typ_constructors_of_typ (typ : t) : Name.Set.t =
   | ForallModule (_, param, result) ->
     local_typ_constructors_of_typs [param; result]
   | ForallTyps (_, typ) | FunTyps (_, typ) -> local_typ_constructors_of_typ typ
-  | Error _ -> Name.Set.empty
+  | _ -> Name.Set.empty
```

Comment:
I would keep an explicit pattern (not `_`) so that we are forced to update this function when there are new constructors.